samples/dotnet/ConsoleConsumer/LocalSchemaRegistry.cs

```diff
@@ -44,26 +41,37 @@ public Task<List<string>> GetAllSubjectsAsync()
             throw new System.NotImplementedException();
```

Comment:
We don't need to throw here. This was introduced earlier.

Reply:
I removed this file. :)